etfl/core/genes.py

```diff
@@ -77,7 +78,27 @@ def transcribed_by(self):
     @transcribed_by.setter
     def transcribed_by(self,value):
         # TODO: Make this a setter that rewrites the adequate constraints
-        raise NotImplementedError()
+        if value != self._transcribed_by:
+            if self.model is None:
+                # Easy
+                self._transcribed_by = value
+            else:
+                # We need to make the model change the corresponding cstr
+                mod_id = self.id + '_transcription'
+                Cstr = self.model.get_constraints_of_type(SynthesisConstraint)
+                cstr_exist = 1
+                try:
+                    Cstr.get_by_id(mod_id)
+                except KeyError:
+                    cstr_exist = 0
```

Comment:
Please use `True` and `False` instead of `1` and `0`

Reply:
Also please add comments as to why you are doing this: checking that the genes are not transcribed yet.